src/main/java/io/zeebe/monitor/rest/ServiceStatusViewController.java

```diff
@@ -0,0 +1,62 @@
+package io.zeebe.monitor.rest;
+
+import io.camunda.zeebe.client.api.response.BrokerInfo;
+import io.camunda.zeebe.client.api.response.PartitionInfo;
+import io.camunda.zeebe.client.api.response.Topology;
+import io.zeebe.monitor.rest.dto.BrokerDto;
+import io.zeebe.monitor.rest.dto.PartitionInfoDto;
+import io.zeebe.monitor.rest.dto.StatusDto;
+import io.zeebe.monitor.zeebe.ZeebeStatusService;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.boot.info.BuildProperties;
+import org.springframework.data.domain.Pageable;
+import org.springframework.stereotype.Controller;
+import org.springframework.web.bind.annotation.GetMapping;
+
+import java.util.Map;
+
+@Controller
+public class ServiceStatusViewController extends AbstractViewController {
+
+  @Autowired private ZeebeStatusService zeebeStatusService;
+  @Autowired BuildProperties buildProperties;
+
+  @GetMapping("/views/service-status")
+  public String index(final Map<String, Object> model, final Pageable pageable) {
+    addDefaultAttributesToModel(model);
+
+    final Topology topology = zeebeStatusService.getTopology();
+    model.put("status", toDto(topology, buildProperties));
+
+    return "service-status-view";
+  }
+
+  private StatusDto toDto(Topology topology, BuildProperties buildProperties) {
+    final StatusDto statusDto = new StatusDto();
+    statusDto.setClusterSize(topology.getClusterSize());
+    statusDto.setGatewayVersion(topology.getGatewayVersion());
+    statusDto.setPartitionsCount(topology.getPartitionsCount());
+    statusDto.setReplicationFactor(topology.getReplicationFactor());
+    statusDto.setSimpleMonitorVersion(buildProperties.getVersion());
```

Comment:
will do.

Reply:
> ZeebeSimpleMonitorApp.class.getPackage().getImplementationVersion()
I was not able to make this proposed solution work, since I have not found a smart way to make the version string available to this Java mechanics.
What I build instead: I did extend the POM so that a META-INF.MF file is created during compile time.
So it will be bundled later in the Docker.
Please keep in mind, with JIB we don't build fat jars, which is cool, but need this manifest for proper version information.